pages/contact.tsx

```diff
@@ -0,0 +1,328 @@
+  const [email, setEmail] = useState('')
+  const [message, setMessage] = useState('')
+  /*
+const res = await fetch(` https://samenvvvv.com/email`, {
+            method: 'POST',
+            body: JSON.stringify({
+              to: 'info@samenvvv.nl',
+              from: "no-reply@samenvvvv.com",
+              replyTo: email,
+              subject: 'CalculatAR Report',
+              html: "<p>Here is example html</p>",
+            text:message,
+            headers: {
+              'Content-Type': 'application/json',
+            },
+          }),
+        })
+*/
+  //   Form validation
+  const [errors, setErrors] = useState<any>({})
+
+  //   Setting button text
+  const button_text = t('contact_message_send')
+  const [buttonText, setButtonText] = useState(button_text)
+
+  const [showSuccessMessage, setShowSuccessMessage] = useState(false)
+  const [showFailureMessage, setShowFailureMessage] = useState(false)
+
+  const handleValidation = () => {
+    const tempErrors: any = {}
+    let isValid = true
+
+    if (fullname.length <= 0) {
+      tempErrors['fullname'] = true
+      isValid = false
+    }
+    if (email.length <= 0) {
+      tempErrors['email'] = true
+      isValid = false
+    }
+    if (message.length <= 0) {
+      tempErrors['message'] = true
+      isValid = false
+    }
+
+    setErrors({ ...tempErrors })
+    return isValid
+  }
+
+  //   const [form, setForm] = useState(false);
+  const handleSubmit = async (event: any) => {
+    event.preventDefault()
+
+    const isValidForm = handleValidation()
+
+    if (isValidForm) {
+      const sending = t('contact_field.message_sending')
+      setButtonText(sending)
+      const res = await fetch(` https://samenvvvv.com/email`, {
+        method: 'POST',
+        body: JSON.stringify({
+          to: 'info@samenvvv.nl',
+          from: 'no-reply@samenvvvv.com',
+          replyTo: email,
+          subject: 'CalculatAR Report',
+          html: '<p>Here is example html</p>',
+          text: message,
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        }),
+      })
+
+      const { error } = await res.json()
+      if (error) {
+        //  console.log(error)
+        setShowSuccessMessage(false)
+        setShowFailureMessage(true)
+        setButtonText(button_text)
+
+        // Reset form fields
+        setFullname('')
+        setEmail('')
+        setMessage('')
+        //   setSubject('')
+        return
+      }
+      setShowSuccessMessage(true)
+      setShowFailureMessage(false)
+      setButtonText(button_text)
+      // Reset form fields
+      setFullname('')
+      setEmail('')
+      setMessage('')
+      //setSubject('')
+    }
+    /// console.log(fullname, email, message) //subject
+  }
+
+  return (
+    <Layout>
+      <Container
+        bg="#9DC4FB"
+        maxW="full"
+        mt={0}
+        centerContent
+        overflow="hidden"
+      >
+        <Flex>
+          <Box
+            bg="#02054B"
```

Comment:
In terms of the consistency of design, it's always better to use theme colors instead of arbitrary colors. Let's consider about creating some theme colors, dark mode designs etc to make it more consistent.